geomesa-web/geomesa-web-csv/pom.xml

```diff
@@ -15,29 +15,93 @@
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
             <artifactId>geomesa-web-core-accumulo1.5</artifactId>
-            <version>1.0.0-rc.4-SNAPSHOT</version>
+            <version>${project.parent.version}</version>
```

Comment:
I think it's better to use the variable, that way they can't get out of sync (this happened in the schema changes branch).